app/renderer/components/common/messageBox.js

```diff
@@ -3,20 +3,34 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const React = require('react')
-const ImmutableComponent = require('../immutableComponent')
+const {StyleSheet, css} = require('aphrodite')
```

Comment:
OK, I understand. I proposed it so we could remove styles in the existing LESS files at the same time. It is fine to replace them with later.

Otherwise: I'm thinking we would set `no-important` to the general components (such as ones in `common` folder) and not to other specific ones, considering [the possibility of removing !important from everything](https://github.com/Khan/aphrodite#todo).

That means:

- `aphrodite` for files in `app/renderer/components/` such as `releaseNotes.js`
- `aphrodite/no-important` for files in `app/renderer/components/common/` such as `dropdown.js`

What do you guys think? @bsclifton @cezaraugusto @NejcZdovc. By doing so we could avoid overwriting `!important` with `!important`, which is obviously not a good habit on CSS.

The thing is that I noticed that it seemed to be impossible to cascade styles with `cx` as we usually do with CSS, which I noticed during refactoring `modalOverlay.js`. eg:

````
<div className={cx({
    [modalOverlay__generalHeaderStyles]: true,
    [css(styles.modalOverlay__paymentHistoryHeaderStyles)]: true
})}
````
I thought that kind of styling should work, overwriting the styles specified with `modalOverlay__generalHeaderStyles`, but I could not find the proper way, instead of creating another wrapper `div` for that `div`. If any, please let me know. @bsclifton @cezaraugusto @NejcZdovc 

We could discuss the thing on another issue if you wish.